en/advanced/parameters_and_configurations.md

```diff
@@ -330,6 +333,26 @@ The purpose of each line is given below (for more detail see [module_schema.yaml
  */
 ```
 
+#### Publishing Parameter Metadata to a GCS
+
+Parameter metadata is collected into a JSON or XML file during each PX4 build.
+For all targets with enough FLASH available the JSON file is xz-compressed and stored within the generated binary.
+The file is then shared to ground stations using the [MAVLink Component Information Protocol](https://mavlink.io/en/services/component_information.html).
+This ensures that parameter metadata is always up-to-date with the code running on the vehicle.
+
+Binaries for flight controller targets with constrained memory (`CONSTRAINED_MEMORY`) do not store the *build-specific* parameter metadata in the binary, but instead reference metadata stored on `px4-travis.s3.amazonaws.com`.
+Versions of the metadata exist for the current master and each PX4 release.
```

Comment:
@bkueng Note, this is my **assumption** - that each release will be adjusted to find its own appropriate metadata. Otherwise you'd be screwed as time goes by and releases get older. Is this correct?

Reply:
Correct